### DISCUSSION TITLE
Better BibTeX download says the file is corrupted?

### DISCUSSION THREAD

**Discussion by cstarrcstarr** [retorquere/zotero-better-bibtex#2777] [Q&A] [answered]
When I download it on Windows, I get a message saying the file cannot be opened because it is corrupted. Is there another download site? Link? Help?

**Comment by retorquere** [comment, marked as answer]
Maybe this? https://retorque.re/zotero-better-bibtex/support/faq/#add-on-could-not-be-installed-because-it-appears-to-be-corrupt

**Reply by cstarrcstarr** [reply]
Thank you! I was thrown because when you right-click it doesn't say "save as" it says "save as a link", but that worked!
